src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Properties/Resources.resx

```diff
@@ -396,4 +396,7 @@
   <data name="UpdatedResourceGroup" xml:space="preserve">
     <value>Updated resource group '{0}' in location '{1}'</value>
   </data>
+  <data name="OperationFailedWithTimeOut" xml:space="preserve">
+    <value>Operation failed because a request time out.</value>
```

Comment:
There isn't any confusion. all good

---
In reply to: [66567017](https://github.com/Azure/azure-powershell/pull/2412#discussion_r66567017) [](ancestors = 66567017)